deltachat-ios/MessageKit/Layout/TextMediaMessageSizeCalculator.swift

```diff
@@ -81,13 +81,22 @@ open class TextMediaMessageSizeCalculator: MessageSizeCalculator {
 
             var messageContainerSize = CGSize(width: imageWidth, height: imageHeight)
             switch message.kind {
-            case .photoText(let mediaItem), .videoText(let mediaItem), .animatedImageText(let mediaItem):
+            case .photoText(let mediaItem), .animatedImageText(let mediaItem):
                 if let text = mediaItem.text?[MediaItemConstants.messageText] {
                     let textHeight = text.height(withConstrainedWidth: maxTextWidth)
                     messageContainerSize.height += textHeight
                     messageContainerSize.height +=  self.messageLabelInsets(for: message).vertical
                 }
                 return messageContainerSize
+            case .videoText(let mediaItem):
+                var videoContainerSize = CGSize(width: self.minTextWidth, height: self.minTextWidth)
```

Comment:
This line looks suspicious to me. 
The layout of the text image cell is not correct right now.
![Screen Shot 2020-06-18 at 16 24 19](https://user-images.githubusercontent.com/2614900/85032787-84ca4480-b180-11ea-94e0-5b378a836ea8.png)

Reply:
The Cooooool is within the picture. You can reproduce it if you share a video + text to a chat

Reply:
Cool thanks, it should be fixed now.